### DISCUSSION TITLE
无法安装4.6.33.0之后更新版本

### DISCUSSION THREAD

**Discussion by ADudu-L** [ywmoyue/biliuwp-lite#855] [Q&A] [answered]
### Bug 描述

4.6.33.0之后更新的版本在安装时均提示安装失败。

### 操作步骤

```plain text
安装证书后双击msix安装包，安装框架的进度条刚出现，就迅速跳转显示安装失败页面。
```

### 期望结果

如何安装最新版本？

### 截图

![Uploading image.png…]()

### 应用来源

Github/其他Github镜像站

### 代码贡献

我更擅长发现问题

**Comment by ywmoyue** [comment, marked as answer]
请查看安装教程 https://github.com/ywmoyue/biliuwp-lite/blob/master/document/install-readme.md

![image](https://github.com/user-attachments/assets/ed6b478d-67dc-4255-9ac5-d33438b140a4)

**Reply by ADudu-L** [reply]
按教程进行到这一步时不知道怎么继续了
![image](https://github.com/user-attachments/assets/c5f61dda-9719-4833-8647-934aea498e07)

**Reply by ywmoyue** [reply]
下一步就是运行安装脚本了

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/blob/master/document/install-readme.md#%E8%BF%90%E8%A1%8C%E5%AE%89%E8%A3%85%E8%84%9A%E6%9C%AC

**Reply by ADudu-L** [reply]
安装成功了，但是打开之后是空白的，也无法登录。
![2](https://github.com/user-attachments/assets/01bd5655-6f7c-4254-8218-5c855ed4df37)

**Reply by ywmoyue** [reply]
如果开了代理的话需要解除uwp回环代理限制

**Reply by ADudu-L** [reply]
我发现是开了vpn的缘故，关掉之后可以正常使用了。谢谢！

**Reply by ywmoyue** [reply]
v2rayn和clash for windows较新版本都有自带这个工具
![image](https://github.com/user-attachments/assets/46533eb7-ca07-471a-bf50-51ef3922985a)

使用方式可以查看app的帮助页的链接

![image](https://github.com/user-attachments/assets/1c372900-67dd-4766-9226-174be5e763b5)
